cypress/page-objects/signInPage.js

```diff
@@ -0,0 +1,82 @@
+import { BasePage } from './basePage';
+
+class SignIn extends BasePage {
+
+  navigateToSignInPage() {
+    cy.visit('/signing');
+  }
+
+  get signInEmailField() {
+    return cy.get('input[data-test="sign-in-email"]');
+  }
+
+  enterSignInEmail(email) {
+    this.signInEmailField.type(email);
+  }
+
+  get signInPasswordField() {
+    return cy.get('input[data-test="sign-in-password"]');
+  }
+
+  enterSignInPassword(password) {
+    this.signInPasswordField.type(password);
+  }
```

Comment:
getters first, then class methods, lets please do not make mess in PO files

Reply:
done